src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          class="logo_img"
+          src="images/logo.png"
+          alt="Mo-yo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_ul">
+          <li class="nav_li">
+            <a
+              href="index.html"
+              class="nav_a is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="index.html"
+              class="nav_a"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="index.html"
+              class="nav_a"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_li">
+            <a
+              href="index.html"
+              class="nav_a blue"
+              data-qa="hover"
+            >
+              Laptops & Computers
```

Comment:
Typographical error: 'Laptops & Computers' should be 'LAPTOPS & COMPUTERS' to match the expected format.